packages/kbn-journeys/journey/journey.ts

```diff
@@ -33,6 +34,7 @@ export interface BaseStepCtx {
   inputDelays: InputDelays;
   kbnUrl: KibanaUrl;
   kibanaServer: KibanaServer;
+  esArchiver: EsArchiver;
```

Comment:
I had a talk with @wayneseymour about capabilities that should be available in journey `step` and we think `esArchiver` is not the one for few reasons:

- since data ingestion affects ES work, Kibana performance metrics won't be accurate. For this reason we load data before journey steps and unload after steps are executed.
- when archives are passed as journey object parameters, we have a full control on it. Things will be more tricky with loading/unloading data during steps and we expect tests to be more flaky.

Reply:
At the moment we suggest journeys design to be very simple and straight forward, let's keep it this way until we hear requests from Teams.

Reply:
Agree, I removed this change.